rust_src/src/keymap.rs

```diff
@@ -19,22 +20,24 @@ use crate::{
     lisp::LispObject,
     lists::{nth, setcdr},
     lists::{LispCons, LispConsCircularChecks, LispConsEndChecks},
+    marker::marker_position,
     multibyte::LispStringRef,
     obarray::intern,
     remacs_sys::{
```

Comment:
Things like `call2` and `list2` are almost always the wrong thing in the Rust code. There is usually a function or a macro to mimic the C code's needs.